popup.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable max-len */
+//     const seeItems = document.querySelector('.card-btn');
+//     const cardsContainer = document.querySelector('.pop');
+//     const section = document.createElement('div');
+//     section.className = 'Popup';
+
+//     const popups = [
+//       {
+//         src: 'images/61.png',
+//         title: 'Tonic',
+//         description1: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor.',
+//         description2: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor.',
+//       },
+//       {
+//         src: 'images/4.png',
+//         title: 'Tonic',
```

Comment:
- [ ] I can see that this whole file is filled with commented codes. Please remove it if you don't need it because It is useless and it contains commented codes 🙏